plugins/modules/source_control/gitlab/gitlab_project_variable.py

```diff
@@ -55,6 +55,44 @@
         See GitLab documentation on acceptable values for a masked variable (https://docs.gitlab.com/ce/ci/variables/#masked-variables).
     default: {}
     type: dict
+  variables:
```

Comment:
maybe we should mark the `vars` parameter as deprecated and remove it in 6.0.0? @felixfontein

Reply:
I'd only deprecate it after the new parameter has been around for some time. This gives users a bit of time to migrate without having to see deprecation warnings all the time (think of playbooks that have to run with different versions of the collections).